web/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/web/rest/JournalforingController.java

```diff
@@ -33,18 +34,23 @@
 
     private final SakerService sakerService;
     private final Tilgangskontroll tilgangskontroll;
+    private final UnleashService unleashService;
+    private static final String SAK_FEATURE = "modiabrukerdialog.rest-sak-impl";
+
 
     @Autowired
-    public JournalforingController(SakerService sakerService, Tilgangskontroll tilgangskontroll) {
+    public JournalforingController(SakerService sakerService, Tilgangskontroll tilgangskontroll, UnleashService unleashService) {
         this.sakerService = sakerService;
         this.tilgangskontroll = tilgangskontroll;
+        this.unleashService = unleashService;
     }
 
     @GetMapping("/saker/sammensatte")
     public List<Sak> hentSammensatteSaker(@PathVariable("fnr") String fnr) {
         return tilgangskontroll
                 .check(tilgangTilBruker.with(fnr))
-                .get(Audit.describe(READ, Person.GsakSaker, new Pair<>(AuditIdentifier.FNR, fnr)), () -> sakerService.hentSammensatteSaker(fnr));
+                .get(Audit.describe(READ, Person.GsakSaker, new Pair<>(AuditIdentifier.FNR, fnr)),
```

Comment:
Tror jeg ville vurdert å legge til ett nytt endepunkt istedet. 
E.g `@GetMapping("/saker")` som returnerte hele `SakerService.Resultat` objektet fra `hentSaker`.

Grunnen er at `hentSaker` og `hentSammensatteSaker` ikke returnerer det samme, og vi må evt håndtere duplikate pensjonssaker i frontend om vi skur på feature-togglen sånn det står nå.

Reply:
Jeg har testet den koden i Q1 miljø vi får samme resultat som `hentSammensatteSaker` fordi at ekskludering av saker er allerede håndtert i frontend. 
Men jobber på frontend koden og kommer med PR
```
function getSaker(
    gsak: AsyncResult<Array<JournalforingsSak>>,
    psak: AsyncResult<Array<JournalforingsSak>>
): JournalforingsSak[] {
    const psakData = hasData(psak) ? psak.data : [];
    const gsakData = hasData(gsak) ? gsak.data : [];

    const psakIder = psakData.map(sak => sak.fagsystemSaksId);
    const gsakSaker = gsakData.filter(sak => !psakIder.includes(sak.fagsystemSaksId));

    return [...gsakSaker, ...psakData];
}
```

Reply:
Duplikate saker var nok ett dårlig eksempel fra min side, men poenget om at kallene ikke returnerer det samme data-messig eller semantisk er fortsatt gyldig. 

Feilsituasjoner er ett sånn eksempel;  
Med feature-toggle `off` så betyr `200 OK` at vi fikk hentet ut data fra `sak`, mens alle andre statuser betyr at det skjedde en feil og vi viser feilmeldingen "*Feil ved uthenting av saker fra GSAK*".

Med feature-toggle `on`, så kan vi få `200 OK` uten å få noen saker fordi alle kallene feilet, og alle feilene blir samlet opp i `SakerService.Resultat`. For å kunne vise disse differensierte feilmeldingene er det derfor viktig at hele `SakerService.Resultat` blir sendt til frontend. 


Andre fordeler med å lage det som eget endepunkt;
- man kan se på om url-pathen kan forenkles. "sammensatte" er ett konsept vi ikke trenger noe mer, så kjekt om vi kan kvitte oss med det.
- man kan kvitte seg med duplikat-sjekken i frontend